src/theme/components/statusBadge.ts

```diff
@@ -39,45 +38,45 @@ const high = defineStyle({
   },
 });
 // Type variants
-const neutral = defineStyle({
+export const neutral = defineStyle({
```

Comment:
Exporting to use in `banner.ts`. I think this can be much cleaner but this works for now.

Reply:
Can these styles be moved into a global space?

Reply:
yes, but later?

Reply:
Sure, but let's not forget about this.  Perhaps this can be tackled when as part of the big "color" shift.